pkg/controller/deployment/deployment_controller.go

```diff
@@ -796,5 +797,6 @@ func (dc *DeploymentController) checkForProgress(key string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
+	glog.V(2).Infof("Syncing deployment %q for a progress check", key)
```

Comment:
https://github.com/kubernetes/kubernetes/issues/39785 flakes only on aws and gke and thank god I can get back controller manager logs in v2 from aws.